addon/components/frost-date-time-picker.js

```diff
@@ -25,6 +29,9 @@ export default Component.extend({
     date: PropTypes.EmberComponent,
     time: PropTypes.EmberComponent,
     value: PropTypes.string.isRequired,
+    dateFormat: PropTypes.string,
+    timeFormat: PropTypes.string,
+    dateTimeFormat: PropTypes.string,
```

Comment:
This is the format of the date sent back to the consumer. Do we really want to give the consumer the possibility to set this? if yes I think we need to have a clearer name to make sure that this concept is clear. If we are keeping the name as is I will think that this is a way to specify the date and time format at once.